### PR TITLE
[MID-15] add deploy possibility & instruction

### PR DESCRIPTION
**Pull request by alex-popov-tech**
https://trello.com/c/pFn53t74/15-feature-add-deploy-possibilities

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,8 +1,9 @@
 NODE_ENV="development"
 
-DATABASE_URL="postgresql://developer:dev_pass@localhost:5432/mayak?schema=public"
```
Comment (epaminond) [inline .env.example:3]: I'd rather keep this kind of placeholder to make setup easier.

File: .env.example [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
+DATABASE_URL=""
+DIRECT_DATABASE_URL=""
 
-ADMIN_USERNAME="admin"
-ADMIN_PASSWORD="admin"
+ADMIN_USERNAME=""
+ADMIN_PASSWORD=""
```
Comment (epaminond) [inline .env.example]: If it's empty, would fail to start an app or would it allow empty login and password? )

Comment (alex-popov-tech) [reply .env.example]: it will fail to start

Comment (alex-popov-tech) [reply .env.example]: actually its probably fail on admin route, rather than just fail everything

**Review by epaminond — DISMISSED** [review]

**Review by epaminond — DISMISSED** [review]

File: .env.example [hunk 3]
```diff
@@ -1,8 +1,9 @@
 NODE_ENV="development"
 
-DATABASE_URL="postgresql://developer:dev_pass@localhost:5432/mayak?schema=public"
+DATABASE_URL="postgres://username:pass@host:5432/database"
```
Comment (BohdanLeshchyshyn) [inline .env.example:3]: Maybe will be better - postgresql://postgres:postgres@localhost:5432/mayak as a default url.
Usually, when you install Postgres it already has the user "postgres" and the default password for such a user is also "postgres".

File: vercel.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "buildCommand": "npm run build",
+  "devCommand": "npm run dev",
+  "installCommand": "npm install"
+}
```
Comment (BohdanLeshchyshyn) [inline vercel.json:5]: It seems you missed an empty line at the end of the file

File: src/app/styles.module.scss
Comment (BohdanLeshchyshyn) [inline src/app/styles.module.scss:1]: I think we will need to remove all unnecessary styles and pages,  but it will be separate task

**Review by Mykhailo-Pasternak — DISMISSED** [review]
good job 👍

**Review by BigTako — APPROVED** [review]

**Review by Mykhailo-Pasternak — APPROVED** [review]

**Pull request merged**